### PR TITLE
Export 2018 08 30 (Canlaon) (Revised)

### PR DESCRIPTION
**Pull request by mTwTm**
l10n for Rocket Canlaon
See here for the [schedule](https://github.com/mozilla-tw/Rocket/wiki) of the release.
Related issues on Rocket:
[Show "... wants to access your location" message too many times when searching](https://github.com/mozilla-tw/Rocket/issues/1654) - [1](https://github.com/mozilla-tw/Rocket/pull/2518/files#diff-01dafb3fd0217441330103cfc8300094) new string.

### CODE REVIEW

**Comment by mTwTm** [comment]
Per [discussion](https://github.com/mozilla-tw/Rocket/issues/2517), using `Don't ask again for this site until %1$s restarts` instead. Deprecates #24

File: locales/templates/app.pot [hunk 1]
```diff
@@ -495,6 +495,16 @@ msgid ""
...
 msgstr ""
 
+#. A checkbox option in the dialog which ask users to give geolocation
+#. permission to the site.         Users' decision will be remembered and
+#. future display of the dialog will be skipped if this         option is
+#. checked. These memorized decisions will be wiped out if the App is
+#. restarted.         %1$s is the App name (Firefox Rocket).
+#, c-format
+msgctxt "geolocation_dialog_message_cache_it"
+msgid "Don't ask again for this site until %1$s restarts"
```
Comment (Delphine) [inline locales/templates/app.pot]: Proper quotes are smart quotes for strings: ’

Comment (mTwTm) [reply locales/templates/app.pot]: I think we already have this one in `my_shot_on_boarding_category_message`:https://pontoon.mozilla.org/id/firefox-rocket/app.po/?search=my_shot_&string=189499 seems fine on Pontoon?

Comment (Delphine) [reply locales/templates/app.pot]: Ouch. That means it fell through the cracks. We should fix that, if possible (can be in a later export, we just need to remember about it)

Comment (mTwTm) [reply locales/templates/app.pot]: I can do it since we're going to fix it anyway :)

Comment (mTwTm) [reply locales/templates/app.pot]: Given it's not a blocking change... :P

Comment (mTwTm) [reply locales/templates/app.pot]: I'll fix both of them now.

**Pull request merged**